warehouse/templates/index.html

```diff
@@ -0,0 +1,52 @@
+{% extends "base.html" %}
+
+{% block content %}
+<section class="index-page">
+    <header class="header">
+        <div class="flash-notice">
+            <p><strong class="site-name hidden-xs hidden-sm">Warehouse (pypi.python.org)</strong>
+            is a repository of software for the Python programming language.</p>
+            <span>For up to date best practices about packaging read
+            <a href="https://packaging.python.org/en/latest/">packaging.python.org</a></span>.
+        </div>
+        
+        <form class="search-bar" role="search" action="TODO" method="GET">
+          <input type="search" placeholder="Search packages ..." />
+          <button type="submit">
+            <img src="{{ request.static_path('warehouse:static/images/search-icon.png') }}" alt="Search Icon">
+          </button>
+        </form>
+        
+        <div class="stats">
+          <ul>
+            <li>{{ num_projects }}<span>Projects</span></li>
+            <li>{{ num_releases }}<span>Releases</span></li>
+            <li>{{ num_files }}<span>Files</span></li>
+            <li>{{ num_users }}<span>Users</span></li>
+          </ul>
+        </div>
+    </header>
+
+    <div class="content">
+        <h1>{{ _('Recently Released') }}</h1>
+        <ul>
+            {% for release in latest_updated_releases %}
+            <div class="comment">
+              <div class="comment-image">
+                    <img src="{{ gravatar(release.author_email, size=50) }}" alt="{{ release.author}}" height="50" width="50">
+              </div>
+              <div class="comment-content">
+                <h1><a href="{{ request.route_path('packaging.project', name=release.project.normalized_name) }}">
+                      {{ release.project.name }}</a>
+                </h1>
+                <p>{{ release.summary }}</p>
```

Comment:
I think `release.summary` can be empty. Wrapping it with an `if' (just like in templates/packaging/detail.html) would be nice.